test/test_t8code_2d.jl

```diff
@@ -30,6 +30,12 @@ mkdir(outdir)
     end
 end
 
+@trixi_testset "test load mesh from path" begin
+    @test_throws "Unknown file extension: .unknown_ext" begin
+        mesh = T8codeMesh(touch("dummy.unknown_ext"), 2)
+    end
```

Comment:
I'd rather have tests not create files outside of elixirs. Maybe you can put this in a temp dir?
```suggestion
    mktempdir() do path
        @test_throws "Unknown file extension: .unknown_ext" begin
            mesh = T8codeMesh(touch(joinpath(path, "dummy.unknown_ext")), 2)
        end
    end
```

Reply:
Done.